src/demo/share/jfc/Font2DTest/Font2DTest.java

```diff
@@ -1021,17 +1429,23 @@ public static void main(String[] argv) {
             }
         }
 
-        UIManager.put("swing.boldMetal", Boolean.FALSE);
-        final JFrame f = new JFrame( "Font2DTest" );
-        final Font2DTest f2dt = new Font2DTest( f);
-        f.addWindowListener( new WindowAdapter() {
-            public void windowOpening( WindowEvent e ) { f2dt.repaint(); }
-            public void windowClosing( WindowEvent e ) { System.exit(0); }
-        });
-
-        f.getContentPane().add( f2dt );
-        f.pack();
-        f.setVisible(true);
+        buildFontInfo();
+        try {
+           UIManager.setLookAndFeel(new NimbusLookAndFeel());
+            SwingUtilities.invokeAndWait(() -> {
+                final JFrame f = new JFrame( "Font2DTest" );
+                final Font2DTest f2dt = new Font2DTest( f);
+                f.addWindowListener( new WindowAdapter() {
+                    public void windowOpening( WindowEvent e ) { f2dt.repaint(); }
+                    public void windowClosing( WindowEvent e ) { System.exit(0); }
```

Comment:
@prrace Minor Spacing issue. I see this pattern in the original code as well, could be a quick find+replace all fix for opening and closing brace. But then again it would also create a lot of git diff footprint along with the fix.

```suggestion
                f.addWindowListener( new WindowAdapter() {
                    public void windowOpening(WindowEvent e) { f2dt.repaint(); }
                    public void windowClosing(WindowEvent e) { System.exit(0); }
```

Reply:
There's a LOT of such things, that my itchy fingers wanted to fix, but  I deliberately resisted temptation and don't see any reason to pick on a few random infractions to fix.

Reply:
Agreed.

Reply:
Yeah, agreed on this.